### ISSUE TITLE
"Tempo Cheat Sheet" saying "Tempo is a trace id lookup store", outdated?

### ISSUE THREAD

**Issue by cyrille-leclerc** [grafana/tempo#2357]
The Tempo Cheat Sheet says

> Tempo is a trace id lookup store. Enter a trace id in the above field and hit “Run Query” to retrieve your trace. Tempo is generally paired with other datasources such as Loki or Prometheus to find traces.
> Here are some [instrumentation examples](https://grafana.com/docs/tempo/latest/guides/instrumentation/) to get you started with trace discovery through logs and metrics (exemplars).

Isn't it an outdated presentation of Tempo? shouldn't we say that "Tempo distributed traces store accessible thought a rich query language: TraceQL

![image](https://user-images.githubusercontent.com/459691/233494412-12bcc520-d448-4041-8d29-f4440424cdbd.png)

I couldn't find where this text is located in the source code so I couldn't initiate a conversation on a pull request.

**Comment by joe-elliott** [comment]
Yeah, we should fix this. That text is old. Moving this to the Grafana repo where that text is controlled.
